hazelcast/src/main/java/com/hazelcast/map/impl/operation/AddIndexOperationFactory.java

```diff
@@ -16,6 +16,7 @@
 
 package com.hazelcast.map.impl.operation;
```

Comment:
Since there are so many map specific OperationFactories, I think it would be practical to introduce an AbstractMapOperationFactory that already takes care of the IdentifiedDataSerializable implementation and getFactoryId. This way we keep the noise down.